draft-ietf-quic-http.md

```diff
@@ -430,29 +469,79 @@ The flags defined are:
 
 The PRIORITY frame payload has the following fields:
 
-  Prioritized Stream:
-  : A 32-bit stream identifier for the request stream whose priority is being
-    updated.
+  Prioritized Request:
+  : A 32-bit identifier for a request.  This contains the stream ID of a request
+    stream when the PUSH_PRIORITIZED flag is clear, or a Push ID when the
+    PUSH_PRIORITIZED flag is set.
 
   Stream Dependency:
```

Comment:
Hmm, I think that "dependent request" means the stream that is dependent, so I changed the name.  I'll fix the picture.